docs/getting-started/getting-started-sql-snowflake.md

```diff
@@ -242,25 +244,35 @@ Here we create customers_nations model that’s going to enrich our customers an
 
 To create a new model simply click on the **(1) + Add Model** in the sidebar, when hovering over Models section, or **(2) Create Model** button. A model creation pop-up will show up, with very similar options, as when we defined the seed, available. Enter the **(3) Model Name** and **(4) Model Path** and that’s it. Finally save the model by pressing **(5) OK.**
```

Comment:
The **4.4** section is overall a little difficult to follow. I'd skip some of the details and reduce the amount of content.

Reply:
I agree. Maybe be better to do fewer videos that are slightly longer. I'll sleep on this and keep working on it.

Reply:
I'm thinking of combining the 4 videos into a single video, then keeping the detailed steps below. This way users can try copilot or they can do the manual configurations steps.

Reply:
Done